### DISCUSSION TITLE
Перенос строки

### DISCUSSION THREAD

**Discussion by vldekov** [totumonline/totum-ru-issues-and-discussions#302] [Как это сделать?] [answered]
ch2=:emailSend(to: $fixed_ad63; title: $fixed_ad60; body: $fixed_ad61;from: $fixed_ad62)
fixed_ad60: str`"КР №"+" "+#regnomer`
fixed_ad61: str`"Заключение экперта - "+#zaklyuchenie+", "+$#nl+"Подпись экперта - "+#podpis`
Как сделать правильно перенос строки с использованием оператора $#nl или другим ?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Мы поисследуем это чуть позже. Замените `$#nl` на `<br>`.

```
ch2=: emailSend(to: $fixed_ad63; title: $fixed_ad60; body: $fixed_ad61; from: $fixed_ad62)
fixed_ad60: str`"КР №" ++ #regnomer`
fixed_ad61: str`"Заключение экперта -" ++ #zaklyuchenie,<br>" + "Подпись экперта -" ++ #podpis`

```

**Reply by vldekov** [reply]
fixed_ad61: str`"Заключение экперта -" ++ #zaklyuchenie + "<br>" + "Подпись экперта -" ++ #podpis`
Спасибо, вот так работает!

**Reply by ttm-alexeypolunin** [reply]
В `body` в `emailSend` ожидается `html`, а он переносы строк не считает за значимые знаки — поэтому в этом месте надо использовать `<br>`.
